src/common/components/floating-faq/index.tsx

```diff
@@ -82,201 +83,207 @@ const FloatingFAQ = () => {
+                              </div>
+                            </div>
+                            <OverlayTrigger placement="bottom" overlay={tooltip}>
+                              <Accordion.Toggle
+                                as={Button}
+                                variant="link"
+                                eventKey="1"
+                                className="p-0"
+                              >
+                                <div
+                                  className={`pointer`}
+                                  onClick={() => {
+                                    setExpandedContact(!expandedContact);
+                                  }}
+                                >
+                                  <span>
+                                    {expandedContact
+                                      ? chevronUpSvgForSlider
+                                      : chevronDownSvgForSlider}
+                                  </span>
+                                </div>
+                              </Accordion.Toggle>
+                            </OverlayTrigger>
+                          </div>
+                        </div>
+                      </Accordion.Toggle>
+                      <Accordion.Collapse eventKey="1">
+                        <Card.Body className="p-3">
+                          <Form.Group>
+                            <InputGroup size="sm" className="username">
+                              <InputGroup.Prepend>
+                                <InputGroup.Text>@</InputGroup.Text>
+                              </InputGroup.Prepend>
+                              <Form.Control
+                                autoFocus={true}
+                                required={true}
+                                placeholder={_t("floating-faq.username")}
+                              />
+                            </InputGroup>
+                          </Form.Group>
+                          <Form.Group>
+                            <InputGroup size="sm" className="message">
+                              <Form.Control
+                                as="textarea"
+                                type="text"
+                                autoFocus={true}
+                                required={true}
+                                maxLength={1000}
+                                placeholder={_t("floating-faq.message")}
+                              />
+                            </InputGroup>
+                          </Form.Group>
+                          <Button
+                            className="submit-btn"
+                            variant="primary"
+                            type="submit"
+                            onClick={() =>
+                              window.open(
+                                "mailto:bug@ecency.com?Subject=Reporting issue&Body=Hello team, \n I would like to report issue: \n",
```

Comment:
I guess it should be translated

Reply:
Translation is not necessary, imho... because email comes to us. If we translate this then people start writing in foreign language which will be bit tough for us to understand the issues they experiencing.